code/modules/projectiles/ammo_datums.dm

```diff
@@ -266,7 +266,10 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 		var/obj/projectile/hitscan/main_proj_hitscan = main_proj
 		effect_icon = main_proj_hitscan.effect_icon
 	for(var/i = 1 to projectile_amount) //Want to run this for the number of bonus projectiles.
-		var/obj/projectile/new_proj = new proj_type(loc_override ? loc_override : main_proj.loc, effect_icon)
+		var/atom/used_loc = loc_override ? loc_override : main_proj.loc
+		var/obj/projectile/new_proj = new proj_type(used_loc, effect_icon)
+		// we do this so if we place inside something, we fly out of it instead of hitting it
+		new_proj.hit_atoms += used_loc.contents
```

Comment:
Does this make it impossible to hit things on your turf? Doesn't matter for tank/mechs, but can for mobs

Reply:
couldnt find a change in behaviour based on my testing